src/main/clojure/opa4j/builtins/array.clj

```diff
@@ -25,8 +25,8 @@
       (throw (Exception. (format "arg 1 is not an array"))))
```

Comment:
Can we throw some custom exception like `BuiltinException` rather than `Exception`, so that we don't accidentally catch unexpected stuff?

Reply:
Are the error messages copied from OPA btw?

Reply:
No, but it's a good idea to align both error messages and logs.